internal/pkg/config/config.go

```diff
@@ -53,12 +53,26 @@ func (c Config) IsSink(call *ssa.Call) bool {
 }
 
 func (c Config) IsSinkFunction(f *ssa.Function) bool {
```

Comment:
Much of this code is duplicated from `callMatcher`. I intend to refactor the code once more tests are in place.